src/main.tf

```diff
@@ -9,7 +9,7 @@ resource "azurerm_log_analytics_workspace" "main" {
   name                = var.md_metadata.name_prefix
   resource_group_name = azurerm_resource_group.main.name
   location            = azurerm_resource_group.main.location
-  tags                = var.md_metadata.name_prefix
+  tags                = var.md_metadata.default_tags
 }
 
 resource "azurerm_kubernetes_cluster" "main" {
```

Comment:
Disk encryption set requires CMK setup, which we don't currently support because Azure Key Vault compliance requires network firewalls to block incoming traffic, including cloud API calls.